api/db/create_db.sql

```diff
@@ -2,6 +2,9 @@ CREATE DATABASE IF NOT EXISTS ${DB_NAME} CHARACTER SET utf8 COLLATE utf8_general
 
 CREATE USER IF NOT EXISTS '${WORKBENCH_DB_USER}'@'%' IDENTIFIED BY '${WORKBENCH_DB_PASSWORD}';
 CREATE USER IF NOT EXISTS '${LIQUIBASE_DB_USER}'@'%' IDENTIFIED BY '${LIQUIBASE_DB_PASSWORD}';
+-- In case the users already exist, change their passwords.
```

Comment:
We need this in db-cdr version too .

Reply:
We only need to create the users in one place; they share the same Cloud SQL instance. So I moved all the create user calls to here (adding public as well soon.)

Reply:
I meant the Change password part. Sorry. I highlighted the comment. Github sucks :) It should sho code below the highlight and not above it. Lol.

Reply:
Ok but maybe we need to change password only in one place too :) I guess that is right ...